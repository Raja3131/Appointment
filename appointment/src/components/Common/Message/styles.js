import {StyleSheet} from 'react-native';
import colors from '../../../styles/theme/colors';

export default StyleSheet.create({
  wrapper: {
    height: 42,

    paddingHorizontal: 5,

    paddingVertical: 13,

    marginVertical: 5,
    borderRadius: 4,
    backgroundColor: colors.primary,
  },

  loaderSection: {
    flexDirection: 'row',
  },

  textInput: {
    flex: 1,
    width: '100%',
  },

  error: {
    color: colors.danger,
    paddingTop: 4,
    fontSize: 12,
  },
  text: {
    fontSize: 14,
    color: '#fff',
  },
});
