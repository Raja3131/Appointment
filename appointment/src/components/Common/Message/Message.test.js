import React from 'react';
import renderer from 'react-test-renderer';
import Message from './Message';

test('renders correctly', () => {
  const tree = renderer
    .create(<Message />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
jest.mock('react-native-gesture-handler', () => {});