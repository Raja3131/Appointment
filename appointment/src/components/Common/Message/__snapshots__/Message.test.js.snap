// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessible={true}
  collapsable={false}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "backgroundColor": undefined,
      "borderRadius": 4,
      "height": 42,
      "marginVertical": 5,
      "opacity": 1,
      "paddingHorizontal": 5,
      "paddingVertical": 13,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "justifyContent": "space-between",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#ffffff",
        }
      }
    />
  </View>
</View>
`;
